docs/docs/1.2.0/deployments/multi-zone.md

```diff
@@ -30,10 +30,10 @@ In a multi-zone deployment mode, services will be running on multiple platforms,
 To implement easy service connectivity, Kuma ships with:
 
 * **DNS Resolver**: Kuma provides an out of the box DNS server on every `remote` control plane that will be used to resolve service addresses when estabilishing any service-to-service communication. It scales horizontally as we scale the `remote` control plane.
-* **Ingress Data Plane**: Kuma provides an out of the box `ingress` data plane proxy mode that will be used to enable traffic to enter a zone from another zone. It can be scaled horizontally. Each zone must have an `ingress` data plane deployed. 
+* **Zone Ingress Proxy**: Kuma provides an out of the box `zone-ingress` proxy mode that will be used to enable traffic to enter a zone from another zone. It can be scaled horizontally. Each zone must have an `zone-ingress` proxy deployed. 
 
 :::tip
-An `ingress` data plane proxy is specific to internal communication within a mesh and it is not to be considered an API gateway. API gateways are supported via Kuma's [gateway mode](/docs/1.1.6/documentation/dps-and-data-model/#gateway) which can be deployed **in addition** to `ingress` data plane proxies.
+A `zone-ingress` data plane proxy is specific to internal communication within a mesh and it is not to be considered an API gateway. API gateways are supported via Kuma's [gateway mode](/docs/1.1.6/documentation/dps-and-data-model/#gateway) which can be deployed **in addition** to `zone-ingress` proxies.
```

Comment:
s/data plane proxy/proxy

thinking that otherwise there _might_ be a bit of confusion here because it's a special kind of proxy? (Yeah, original text was not great either ...)